contracts/src/systems/caravan/move_caravan.cairo

```diff
@@ -0,0 +1,64 @@
+#[system]
```

Comment:
this is exactily what we need, one remark though is that we should not link this to the caravans. Caravan is a way to put multiple movable entities in the same entity and move them together. 
But Travel should be for any movable entity, not just caravans. What I would suggest is to:
1. rename this to mod Travel
2. remove any notion of caravan in the code. It just needs to be anything that is movable and has a position.
3. create a folder travel and put it in there

Reply:
i think you're mistaking. The Movable component means anything that can move by itself, not movable goods. So like a horse is movable. An army is movable. Gold is not movable. But it can be moved by a movable entity. 
I agree that "Movable" might be confusing though.
As for only caravans being able to move things. I think that would go against ECS. In our implementation, any entity that has a Capacity component and a Movable component, should be able to move resources around.

Reply:
You're right